src/runtime_json.cpp

```
@@ -754,33 +754,26 @@ int64_t __ry_json_len(void *value) {
-        if (!data[i]) {
-            for (int64_t j = 0; j < i; j++)
-                free((void*)data[j]);
-            free((void*)data);
-            free(header);
-            return nullptr;
-        }
+    std::vector<std::string> keys;
+    keys.reserve(len);
+    for (int64_t i = 0; i < len; i++)
+        keys.emplace_back(v->object_val.keys[i]);
+    auto *result = makeStringList(keys);
```

Clang (14.0.6)

[warning] variable 'result' is not initialized

(cppcoreguidelines-init-variables)